tools/docker/Dockerfile.centos6

```diff
@@ -23,15 +23,19 @@ LABEL Vendor="Apache.org" License="ApacheV2" Version="4.6.0"
 ENV PKG_URL=http://jenkins.buildacloud.org/job/package-rhel63-master/lastSuccessfulBuild/artifact/dist/rpmbuild/RPMS/x86_64
 
 # install CloudStack
-RUN yum install -y \
+RUN yum install -y nc wget \
     ${PKG_URL}/cloudstack-common-4.6.0-SNAPSHOT.el6.x86_64.rpm \
     ${PKG_URL}/cloudstack-management-4.6.0-SNAPSHOT.el6.x86_64.rpm
 
 RUN cd /etc/cloudstack/management; \
     ln -s tomcat6-nonssl.conf tomcat6.conf; \
     ln -s server-nonssl.xml server.xml; \
-    ln -s log4j-cloud.xml log4j.xml
+    ln -s log4j-cloud.xml log4j.xml; \
+    cd /usr/share/cloudstack-common/scripts/vm/hypervisor/xenserver; \
+    wget http://download.cloud.com.s3.amazonaws.com/tools/vhd-util
```

Comment:
Perhaps instead of theses 2 last lines:
wget -O /usr/share/cloudstack-common/scripts/vm/hypervisor/xenserver/vhd-util http://download.cloud.com.s3.amazonaws.com/tools/vhd-util
(one line)